docs/md/public-issues/issue-1.md

```diff
@@ -31,7 +30,7 @@ then we want to help spread the word.
 
 > :Buttons
 >
-> Eugene Ghanizadeh Khoub \
+> Coding Blog Team  \
```

Comment:
customarily this should be the signature of the person on the team who writes the intro, not a generic team signature.